<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadLibraryButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load library</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startSimUnitButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start simulation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="libPathEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>C:/Program Files/SIMULATIONUnit_V9.1Upd1/SIMBAkern64.dll</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="simPathEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>E:/00_SharedFolder/3FL5MB411.spf</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="symbol1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="value1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="symbol2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="value2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
